rules/windows/process_creation/proc_creation_win_addinutil_uncommon_dir_exec.yml

```
@@ -4,8 +4,9 @@ status: test
 description: Detects execution of the Add-In deployment cache updating utility (AddInutil.exe) from a non-standard directory.
 references:
     - https://www.blue-prints.blog/content/blog/posts/lolbin/addinutil-lolbas.html
-author: Michael McKinley (@McKinleyMike), Tony Latteri (@TheLatteri)
+author: Michael McKinley (@McKinleyMike), Tony Latteri (@TheLatteri), Swachchhanda Shrawan Poudel (Nextron Systems)
```

Adding extra path does not make you an author.
```suggestion
author: Michael McKinley (@McKinleyMike), Tony Latteri (@TheLatteri)
```